reconfigure.bash

```diff
@@ -11,7 +11,7 @@ function usage {
 
 BUILDFARM_DEPLOYMENT_PATH=/root/buildfarm_deployment
 BUILDFARM_DEPLOYMENT_URL=https://github.com/ros-infrastructure/buildfarm_deployment.git
-BUILDFARM_DEPLOYMENT_BRANCH=master
+BUILDFARM_DEPLOYMENT_BRANCH=jenkins-lts-upgrades
```

Comment:
Is this indended as a permanent branch?

Reply:
No. We should back it out once https://github.com/ros-infrastructure/buildfarm_deployment/pull/207 and this PR are ready to merge but it's here now for those who would test it.

Reply:
`jenkins-lts-upgrades` branch in [ros-infrastructure/buildfarm_deployment](https://github.com/ros-infrastructure/buildfarm_deployment.git)

instead the `jenkins-lts-upgrade` branch without `s` in the end exists.

I'm going to close #38 due to this is a better place to comment it.

Reply:
Thanks for catching that.